Core/H5PSymfony.php

```diff
@@ -1029,6 +1044,6 @@ private function truncateTable($tableClassName)
      */
     public function libraryHasUpgrade($library)
```

Comment:
maybe implement the typing too ? 
```suggestion
    public function libraryHasUpgrade($library): bool
```